equinox-shipment/Domain/FinalizationTransaction.fs

```diff
@@ -0,0 +1,90 @@
+module FinalizationTransaction
+
+module Events =
+
+    let [<Literal>] CategoryId = "FinalizationTransaction"
```

Comment:
as before, move to top level, rename to `Category`, stop using active pattern for `streamName`

Reply:
Updated thanks!